sdk/communication/AzureCommunicationChat/Source/Generated/AzureCommunicationChatClient.swift

```diff
@@ -70,53 +65,7 @@ public final class AzureCommunicationChatClient: PipelineClient, PageableClient
         )
     }
 
-    public func url(
-        host hostIn: String? = nil,
-        template templateIn: String,
-        pathParams pathParamsIn: [String: String]? = nil,
-        queryParams queryParamsIn: [QueryParameter]? = nil
-    ) -> URL? {
-        var template = templateIn
-        var hostString = hostIn
-        if template.hasPrefix("/") { template = String(template.dropFirst()) }
-
-        if let pathParams = pathParamsIn {
-            for (key, value) in pathParams {
-                if let encodedPathValue = value.addingPercentEncoding(withAllowedCharacters: .azureUrlPathAllowed) {
-                    template = template.replacingOccurrences(of: "{\(key)}", with: encodedPathValue)
-                }
-                if let host = hostString {
-                    hostString = host.replacingOccurrences(of: "{\(key)}", with: value)
-                }
-            }
-        }
-
-        if let hostUnwrapped = hostString,
-            !hostUnwrapped.hasSuffix("/") {
-            hostString = hostUnwrapped + "/"
-        }
-        let urlString = (hostString ?? endpoint.absoluteString) + template
-        guard let url = URL(string: urlString) else {
-            return nil
-        }
-
-        guard !(queryParamsIn?.isEmpty ?? false) else { return url }
-
-        return appendingQueryParameters(url: url, queryParamsIn ?? [])
-    }
-
-    private func appendingQueryParameters(url: URL, _ queryParams: [QueryParameter]) -> URL? {
-        guard !queryParams.isEmpty else { return url }
-        guard var urlComps = URLComponents(url: url, resolvingAgainstBaseURL: true) else { return nil }
-
-        let queryItems = queryParams.map { name, value in URLQueryItem(
-            name: name,
-            value: value?.addingPercentEncoding(withAllowedCharacters: .azureUrlQueryAllowed)
-        ) }
-        urlComps.percentEncodedQueryItems = queryItems
-        return urlComps.url
-    }
+    public lazy var azureCommunicationChatService = AzureCommunicationChatService(client: self)
 
-    public lazy var azureCommunicationChatService: AzureCommunicationChatService =
-        AzureCommunicationChatService(client: self)
+    // MARK: Public Client Methods
```

Comment:
Do we need this comment? Seem like the next link is "}" no method is followed by this line.

Reply:
Good point. There would be public methods if there was no command group, since then the operations would live here. That would be a change in autorest.